tests/arm/Tests.lst

```diff
@@ -83145,7 +83145,7 @@ RelativePath=JIT\Regression\CLR-x86-JIT\V1-M09\b15783\b15783\b15783.cmd
 WorkingDir=JIT\Regression\CLR-x86-JIT\V1-M09\b15783\b15783
 Expected=0
 MaxAllowedDurationSeconds=600
-Categories=EXPECTED_FAIL;13604;Pri1
```

Comment:
How is it connected with legacy backend?

Reply:
ah, I noticed that issue was closed and it seemed the tests should pass now. What we should do is change it to `EXPECTED_PASS` and remove the issue. I'll just leave it as is for now.